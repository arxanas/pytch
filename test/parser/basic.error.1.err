UNEXPECTED_TOKEN[1001] in test/parser/basic.error.1.pytch, line 1, character 5:
Error: I was expecting the end of the file, but instead got an identifier.
   +------------------------------------------------------------------------+
   | test/parser/basic.error.1.pytch                                        |
 1 | foo bar                                                                |
   |     ^~~ Error: I was expecting the end of the file, but instead got an |
   | identifier.                                                            |
   +------------------------------------------------------------------------+
