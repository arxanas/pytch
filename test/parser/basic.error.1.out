SyntaxTree
    IdentifierExpr
        Token 'foo'
    Leading ' bar\n'
    Token EOF ''
